Il2ServerAppLateNight/Il2ServerAppLateNight/src/WaterTemp.cpp

```diff
@@ -18,7 +19,26 @@ std::vector<double> ReadWaterTemps(HANDLE hProcess, LPVOID codeCaveAddress)
 
 		//pointer +178 is offset for water temp in kelvin
 		ReadProcessMemory(hProcess, (LPCVOID)((uintptr_t)targetAddress + 0x178), &rawData, sizeof(double), 0);
+
 		values[i] = *reinterpret_cast<double*>(rawData);
+
+		//Bf 109 K4 Has it's own way to calc water temp
+		std::string v = "Bf 109 K-4";
```

Comment:
Once helper functions for plane string comparison are refactored, use `isBF109K$`